versioned_docs/version-2/platform/integration/bigcommerce/embedded-checkout.md

```diff
@@ -0,0 +1,102 @@
+---
+id: embedded-checkout
+title: Embedded Checkout
+sidebar_label: Embedded Checkout
+description: How to use the BigCommerce Embedded Checkout
+---
+
+## About
+
+The [BigCommerce Embedded Checkout](https://developer.bigcommerce.com/api-docs/storefronts/embedded-checkout/embedded-checkout-tutorial) allows customer to load an iframe version of the checkout. The benefits to this is it supports all the features and payment methods BigCommerce supports out of the box.
+
+## How to use
+
+To use the BigCommerce Embedded checkout you simply need to add it into your checkout components.
+
+With our core app this will be:
+
+`client/src/pages/shop/Checkout/Checkout.js`
+
+### 1. Import the component
+
+```js
+import { EmbeddedCheckout } from './components';
+```
+
+:::note Check the component exists
+Out of the box you should have a `client/src/pages/shop/Checkout/components/EmbeddedCheckout.js` file. If not, please create a new app using `@deity/create-falcon-app` and grab it from there. This is specific to demo-v2
+:::
+
+### 2. Get the embedded checkout URL
+
+Our cart query can have extra params passed to it to grab the embedded checkout iframe URL needed to render the checkout.
+
+Find `CartQuery` in the `Checkout.js` file.
+
+It should look something like this:
+
+```js
+export const CartQuery = props => {
+  return <FalconCartQuery fetchPolicy="cache-and-network" {...props} />;
+};
+```
+
+Change this to add `variables={{ includeEmbeddedUrl: true }}`
+
+```js
+export const CartQuery = props => {
+  return <FalconCartQuery fetchPolicy="cache-and-network" variables={{ includeEmbeddedUrl: true }} {...props} />;
```

Comment:
@hobadams I assume we no longer use `CHECKOUT_USE_EMBEDDED` config var for controlling this?

Reply:
I've not included it, the client can add this to their app if they want.